Umbraco-Cloud/Set-Up/Manage-Hostnames/ca-record-migration.md

```diff
@@ -0,0 +1,27 @@
+# Migrate to new Certificate Authority for custom hostnames
+
+The following changes in Certificate Authority (CA) used to issue certificates for all Umbraco Cloud sites' for new and existing custom hostnames.
+
+## Certificates for new custom hostnames
+
+On September 26th, 2022 certificates will be issued using  'Google Trust Services' instead of 'DigiCert', and Certificate validity will be decreased from 1 year to 90 days.
+
+## Certificates for existing custom hostnames
+
+From October 31st Certificate renewals will no longer use 'DigiCert' as the issuing CA. The renewed certificate will instead be issued by 'Google Trust Services',  and Certificate validity will be decreased from 1 year to 90 days.
+
+No action is required unless you set a Certificate Authority Authorization (CAA) record on your domain.
```

Comment:
This should be moved to the previous section and highlighted either as a note or **bold**. It has to be very clear that no action is required unless you are using CAA

Reply:
Makes sense!